src/routes/accounts.ts

```diff
@@ -117,21 +118,15 @@ export const showPersonAccounts = async (req, res) => {
   res.status(200).send(accounts);
 };
 
-let counter = 0;
-
-export const createAccount = async (personId, data) => {
```

Comment:
This breaks usages of on the backend. Some code imports MockSolaris functions directly, e.g. import MockSolaris from "@kontist/mock-solaris";
And invoke these functions. You can search "MockSolaris.createAccount(" on the backend project to usages of data property.

Reply:
oh... didn't notice it! thanks!